projects/assets/src/translations/en/payment.ts

```diff
@@ -24,6 +24,7 @@ export const payment = {
     expirationMonth: 'Expiration month {{ selected }}',
     useThisPayment: 'Use this payment',
     addNewPayment: 'Add New Payment',
+    paymentAddedSuccessfully: 'New payment was sucessfully added!',
```

Comment:
small typo `sucessfully` should be `successfully`

Reply:
there is some inconsistency in the message format. 

adding new address message reads (existing translation key): New address was added successfully!
adding new payment method message reads (new translation key): New payment was successfully added!

since both of these alert messages could be triggered in close proximity, we should align the verbiage for consistency. 

please update the message to read: `New payment was added successfully!`